### ISSUE TITLE
Module build failed: Error: Cannot find module "../math-constants"

### ISSUE THREAD

**Issue by jimzhan** [ant-design/ant-design#11417]
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Version
3.7.1

### Environment
Windows 7 | Node.js v10.6.0 | React v16.4.1 | ReactDOM v16.4.1

### Reproduction link
[https://github.com/jimzhan/webapp](https://github.com/jimzhan/webapp)

### Steps to reproduce
```bash
git clone https://github.com/jimzhan/webapp
yarn install -V
yarn workspace @webapp/ui build
```

### What is expected?
Build sucessful

### What is actually happening?
Failed to compile.
```bash
C:/Users/jim/workspace/webapp/node_modules/antd/lib/styles/index.less
Module build failed: Error: Cannot find module '../math-constants'
```

---

```javascript
// config-overrides.js

const { injectBabelPlugin } = require('react-app-rewired')
const rewireLess = require('react-app-rewire-less')

module.exports = (config, env) => {
  /* eslint-disable no-param-reassign */
  config = injectBabelPlugin(['import', { libraryName: 'antd', style: true }], config)

  config = rewireLess.withLoaderOptions({
    javascriptEnabled: true,
    modifyVars: {
    /* ------- Override Gloabal Ant Styles ---------- */
    },
  })(config, env)
  /* eslint-enable */

  return config
}
```

**Comment by ojab** [comment]
https://github.com/less/less.js/issues/3295

**Comment by matthew-dean** [comment]
It appears this is a Yarn bug, not a Less bug, see: https://github.com/less/less.js/issues/3295#issuecomment-407098175

**Comment by jimzhan** [comment]
@matthew-dean tks Matthew, I tried upgrade to yarn v1.8 & v1.9, same issue, & it's only happening on Windows (v7 that I'm using).

**Comment by jimzhan** [comment]
Fixed by installing the latest `less@3.8.0`.
